configs/global.go

```diff
@@ -0,0 +1,6 @@
+package configs
+
+const (
+	ExamplePrivateKey = "private_key"
```

Comment:
Don't create a file and folder for example config, please remove it and use the environment variables for example.